webcompat/helpers.py

```diff
@@ -728,12 +728,19 @@ def get_extra_labels(form):
     """Extract extra_labels.
 
     If extra_labels param exists in current session, use it,
-    otherwise use the value coming from form.
+    otherwise use the value coming from form. Add form-v2-experiment
+    label if the experiment cookie exists.
     """
 
     extra_labels = session.pop('extra_labels', None)
 
-    if not extra_labels and 'extra_labels' in form:
+    if not extra_labels and 'extra_labels' in form and form['extra_labels']:
         extra_labels = json.loads(form['extra_labels'])
 
+    if ab_active('exp') == 'form-v2':
+        if isinstance(extra_labels, list):
+            extra_labels.append('form-v2-experiment')
+        else:
+            extra_labels = ['form-v2-experiment']
+
```

Comment:
Changed my initial approach for adding experiment label since I've discovered a bug where the 'form-v2-experiment' label didn't get added if other labels exist.

Reply:
I will check that later today.